src/main/java/com/example/event_manager/enums/SortHowEnum.java

```diff
@@ -0,0 +1,30 @@
+package com.example.event_manager.enums;
+
+import org.springframework.data.domain.Sort;
+
+public enum SortHowEnum {
+  NAME_ASC,LOCATION_ASC,DATE_ASC,TOPIC_ASC,
+  NAME_DSC,LOCATION_DSC,DATE_DSC,TOPIC_DSC;
+
+  public static Sort sortAccordingly(final SortHowEnum sortHowEnum) {
```

Comment:
Instead of this You can have field Sort here and just assign value above

NAME_ASC(Sort.by("name").ascending())